dashboard/sse.go

```diff
@@ -104,7 +79,9 @@ func (emitter *eventEmitter) ServeHTTP(res http.ResponseWriter, req *http.Reques
 
 	res.Header().Set("Access-Control-Allow-Origin", "*")
 
+	emitter.wait.Add(1)
 	emitter.Server.ServeHTTP(res, req)
+	emitter.wait.Done()
```

Comment:
nit: This likely should be in `defer` as `ServeHTTP` can panic, but something later can handle that (net/http does this by defaul - although in this case I think it won't catch it)

Reply:
Oh, you are right, I'll reopen the issue and fix it